addons/cargo/functions/fnc_onOpenInventory.sqf

```diff
@@ -0,0 +1,136 @@
+#include "..\script_component.hpp"
+/*
+ * Author: joko // Jonas, BadGuy
+ * Handles Inventory UI when vehicle has a Cargo Inventory
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Container <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [vehicle] call ace_cargo_fnc_addCargoItem
+ *
+ * Public: No
+ */
+
+if !(GVAR(enable)) exitWith {};
+
+params ["_unit", "_container"];
+
+// This is the 2nd Part of the Hack for Vehicles that dont have an Inventory
+if (_container isKindOf "GroundWeaponHolder") then {
+    private _target = cursorObject;
+    if (!(_target getVariable [QGVAR(hasInventory), true]) && ((ACE_Player distance _target) < MAX_LOAD_DISTANCE) && !lockedInventory _target) then {
+        _container = _target;
+    };
+};
+
+if !(_container getVariable [QGVAR(hasCargo), getNumber (configOf _container >> QGVAR(hasCargo)) == 1]) exitWith {};
+
+GVAR(interactionVehicle) = _container;
+
+GVAR(interactionParadrop) = _container isKindOf "Air" && {
+                                private _turretPath = _unit call CBA_fnc_turretPath;
+                                (_unit == (driver _container)) || // pilot
+                                {(getNumber (([_container, _turretPath] call CBA_fnc_getTurret) >> "isCopilot")) == 1} || // coPilot
+                                {_turretPath in (getArray (configOf _container >> QGVAR(loadmasterTurrets)))} // loadMaster turret from config 
```

Comment:
I didn't know this was a thing. Neat.

Reply:
copied from here: https://github.com/acemod/ACE3/blob/9873bf566ca06f5f88538a216ac0d8fb61f22245/addons/cargo/functions/fnc_initVehicle.sqf#L75-L78 ^^